tests/test_plugins/test_adjoint.py

```diff
@@ -95,7 +95,7 @@ def run_emulated_bwd(
     folder_name: str,
     callback_url: str,
     verbose: bool,
-    num_proc: int = 2,
+    num_proc: int = NUM_PROC_SERVER,
```

Comment:
I had put `num_proc = 2` here and `num_proc = 1` in the `_async` test below so that both parallel and sequential gradient computation are tested (somewhat, as the async tests are not as full). I guess a better way would be to parametrize over one more argument, but do we want to do that?

Reply:
The paramerization over a single run local of true and false already hits both conditions. So this change just means that both conditions are tested for Async too

Reply:
Ohh right.